http/http.ts

```diff
@@ -7,6 +7,7 @@ import * as https from 'https';
 // @ts-ignore
 import * as URLParse from "url-parse";
 import { Observable, from } from '../rxjsStub';
+import * as queryString from "query-string";
```

Comment:
`import { stringify } from "query-string"`

Reply:
When I try to do this, I get this error:

    http/http.ts:10:10 - error TS2614: Module '"query-string"' has no exported member 'stringify'. Did you mean to use 'import stringify from "query-string"' instead?
    
 I can try to use another package instead of query-string since there are issues with this package

Reply:
I found one called @types/qs which has a stringify option that works well without having to edit the jest command, will push this up as part of latest commit